lib/utils.js

```diff
@@ -7,18 +7,18 @@ var changeTypeLookup = {
   abbreviation: abbreviation
 };
 
-var isNullOrUndefined = function(val) {
+exports.isNullOrUndefined = function(val) {
```

Comment:
No. We are not assigning `exports = function(x){};` We are assigning a function to a value within the exports object.
```Javascript
exports = {
  isNullOrUndefined: function(x) {}
}
```